data/json/recipes/food/bread.json

```
@@ -47,5 +47,55 @@
       [ [ "salt", 1 ] ]
     ],
     "//": "Later: this needs baking soda. Also possibility of rewrite to use portions of components' charges."
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "hallula",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_BREAD",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "charges": 16,
```

```suggestion
    "charges": 8,
```
I think this will fix your calorie mismatch in the tests.  If it works change the other one as well.  Otherwise compare the two test failure results and see what direction you need to go on charges.